crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -116,6 +116,16 @@ impl Mesh {
         );
     }
 
+    /// Removes the data for a vertex attribute
+    pub fn remove_attribute(
+        &mut self,
+        attribute: MeshVertexAttribute,
```

Comment:
This should probably follow the `Into<MeshVertexAttributeId>` pattern used elsewhere in Mesh